app/views/vehicles/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Update <%= @vehicle.name %></h1>
+<form action=<%="/vehicles/#{@vehicle.id}"%> method="post">
```

Comment:
I believe we need to use form tag according to project requirements

Reply:
Implementing this in a separate branch for clarity

Reply:
Good idea. I will merge this PR.